torch/quantization/__init__.py

```diff
@@ -10,6 +10,14 @@
 from .quantization_mappings import *  # noqa: F403
 from .fuser_method_mappings import *  # noqa: F403
 
+import warnings
+warnings.warn(
+    "The `torch.quantization` folder has moved to "
+    "`torch.ao.quantization`, please move your callsite over to the new "
```

Comment:
`please move your callsite over to the new location` -- I feel this is a bit confusing, maybe reword this as "please rename torch.quantization to torch.ao.quantization" or something similar to be clearer, and giving a concrete example would be even better